### PR TITLE
Third New England Urbit meetup 

### PR DESCRIPTION
**Pull request by 7Sages7**
Added the .md file for the third New England meetup event page per the conditions of my bounty for UF.

### CODE REVIEW

File: content/events/2024-07-27-Urbit-New-England-Xthirdmeetup.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
++++ title = "Urbit Meetup in Somerville, MA - Aeronaut Brewing" starts = "2024-07-27T14:00:00" ends = "2024-07-27T17:00:00" timezone = "America/New_York" location = "Aeronaut Brewing - 14 Tyler St, Somerville, Massachusetts 02143" image = "https://media.urbit.org/site/ecosystem/communities/new-england.jpg" registration_url = "https://www.meetup.com/urbit-new-england-group/events/302151131/" description = "The third meetup for Urbiters in New England" dark = true darken_image = "true"
+
++++
```
Comment (ripperi) [inline content/events/2024-07-27-Urbit-New-England-Xthirdmeetup.md:13]: These attributes need to be newline delimited. When using existing events as reference, view them in raw mode first. The markdown preview shows the rendered output, not the raw source content

Comment (7Sages7) [reply content/events/2024-07-27-Urbit-New-England-Xthirdmeetup.md:13]: fixed. I'll remember this for next time. Thanks.

**Review by 7Sages7 — COMMENTED** [review]
fixed. I'll remember this for next time. Thanks.

**Comment by 7Sages7** [comment]
Ok. I resubmitted this pull request and incorporated your suggested edit with the delimited lines. Please let me know if you need anything else.

Sent with [Proton Mail](https://proton.me/) secure email.

File: content/events/2024-07-27-Urbit-New-England-Xthirdmeetup.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
++++ title = "Urbit Meetup in Somerville, MA - Aeronaut Brewing"
```
Comment (ripperi) [inline content/events/2024-07-27-Urbit-New-England-Xthirdmeetup.md]: Here as well
```
+++
title = "Urbit Meetup in Somerville, MA - Aeronaut Brewing"
```

Comment (ripperi) [reply content/events/2024-07-27-Urbit-New-England-Xthirdmeetup.md]: Then this should be good to go!

**Comment by 7Sages7** [comment]
Great. I made the fix. Thank you.

Though this is a bit urgent, it's also okay if you are not able to get this on urbit.org until after the event. I just need it on there for the historical record as I am doing this as part of a grant for UF. (All of my RSVPing is done on Landscape and a third-party service, MeetUp, so everything is fine in that regard.).

Sent with [Proton Mail](https://proton.me/) secure email.

**Review by ripperi — APPROVED** [review]

**Pull request merged**